skietbaan_fe/src/App.js

```diff
@@ -2,7 +2,9 @@
 
 import React, { Component } from 'react'
 import { connect } from 'react-redux';
-import './App.scss'
+import './App.scss';
+import {Provider} from 'react-redux';
```

Comment:
Add space between brackets and `Provider`. It's inconsistent with the rest of the code

Reply:
Just auto format. if (VSCode) use cntrl + a + cntrl + k f (cmd on mac)

Reply:
Sorted. Kindly review